samples/mesh/Makefile

```diff
@@ -2,6 +2,6 @@ XBE_TITLE = nxdk\ sample\ -\ mesh
 GEN_XISO = $(XBE_TITLE).iso
 SRCS = $(CURDIR)/math3d.c $(CURDIR)/main.c
 SHADER_OBJS = ps.inl vs.inl
-NXDK_DIR = $(CURDIR)/../..
+NXDK_DIR ?= $(CURDIR)/../..
 
```

Comment:
Might want to delete this empty line in order to unify this Makefile with the majority.